insalan/tournament/models/name_validator.py

```diff
@@ -0,0 +1,86 @@
+"""
+NameValidator class
+"""
+from django.utils.translation import gettext_lazy as _
+from django.db import models
+from insalan.settings import RIOT_API_KEY
+import requests
+
+class EmptyNameValidator:
+    """
+    NameValidator class
+    """
+    short = "None"
+    name = _("Pas de Validation de nom")
+
+    @staticmethod
+    def is_valid(name: str):
+        """
+        This method is used to validate the name of a player
+        Each validators should implement this method
+        """
+        return True
+
+class LeagueOfLegendsNameValidator:
+    """
+    LeagueOfLegendsNameValidator class
+    """
+    short = "LoL"
+    name = _("Validation League of Legends")
+
+    Account_endpoint = "https://europe.api.riotgames.com/riot/account/v1/accounts/by-riot-id/{}/{}?api_key={}"
+    Summoner_endpoint = "https://euw1.api.riotgames.com/lol/summoner/v4/summoners/by-puuid/{}?api_key={}"
```

Comment:
The case doesn't really make sense here

Reply:
It's to not hardcode the API urls. But if you think it's better to put it directly in the code, I can.

Reply:
Yes, this is a good habit, I was talking about the capitalization (lower/upper case). Usually, you'd put constants as all-caps or snake case (without the first capital letter)

Reply:
Oh, yes, mb